### PR TITLE
Update message argument document

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

* Adds defined argument names for DS record digest algorithms and RCODE.
* Updates incomplete text.

## Context

* DS record digest algorithm argument used in #1059.
* RCODE argument used in #1061.

## How to test this PR

Document only, but verify with test case implementations.

### CODE REVIEW

File: docs/logentry_args.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ suffixed *argument name* is not to be listed in this document, it is just an
...
 
 As an example, if two arguments of type "List of IP addresses" are to be used in
-a message, then both *argument names* should `ns_ip_list`. Let us say that one
-is connected to the NSEC record type and the other to the NSEC3 record type. The
-two resulting argument names could then be `ns_ip_list_nsec` and
-`ns_ip_list_nsec3`, respectively.
+a message, then both *argument names* should be `ns_ip_list` following the list
+of defined arguments below. Let us say that one is connected to the NSEC record
+type and the other to the NSEC3 record type. The two resulting argument names
+could then be `ns_ip_list_nsec` and `ns_ip_list_nsec3`, respectively.
```
Comment (tgreenx) [inline docs/logentry_args.md:33]: Here maybe change "Could" to "Should"?

File: docs/logentry_args.md [hunk 2]
```diff
@@ -44,24 +44,28 @@ Example of a message (*msgid* in this case) where this is in use:
...
+| module_list    | List of Zonemaster test modules          | A list of Zonemaster test modules, separated by ":".                    |
+| ns             | Domain name and IP address pair          | The name and IP address of a name server, separated by "/".             |
+| ns_ip          | IP address                               | The IP address of a name server.                                        |
+| ns_ip_list     | List of IP addresses                     | A list of name servers, as specified by "ns_ip", separated by ";".      |
+| ns_list        | List of domain name and IP address pairs | A list of name servers, as specified by "ns", separated by ";".         |
+| nsname         | Domain name                              | The domain name of a name server.                                       |
+| nsname_list    | List of domain names                     | A list of name servers, as specified by "nsname", separated by ";".     |
+| rcode          | An RCODE Name                            | An RCODE Name (not numeric code) from [DNS RCODEs].                     |
```
Comment (tgreenx) [inline docs/logentry_args.md:67]: An -> A

Comment (mattias-p) [reply docs/logentry_args.md:67]: I believe "An RCODE" is correct. I.e. that the rule is based on the pronunciation.

**Comment by mattias-p** [comment]
LGTM, with the change from "could" to "should".

File: docs/logentry_args.md (hunk 1 above)
Comment (matsduf) [reply docs/logentry_args.md:33]: This is an example, so I am not sure that "should" is relevant here, but I guess it would not harm either.

**Comment by matsduf** [comment]
@mattias-p, please rereview.

**Review by mattias-p — APPROVED** [review]
Looking good

**Review by tgreenx — APPROVED** [review]

**Pull request merged**